ads/google/a4a/google-data-reporter.js

```diff
@@ -44,9 +44,9 @@ import {
  * @const {!Object<string,!../../../src/experiments.ExperimentInfo>}
  */
 export const PROFILING_BRANCHES = {
-  a4aProfilingRate: {
-    isTrafficEligible: () => true,
-    branches: ['unused', 'unused'],
+  'a4aProfilingRate': {
```

Comment:
@choumx I understand why we have quotes around `'a4aProfilingRate'`, but not why we have quotes for `'isTrafficEligible'`. The letter should be referenced against `ExperimentInfo` typedef, right?

/cc @erwinmombay